lib/view/events/events_screen.dart

```diff
@@ -75,10 +75,19 @@ class EventsPage extends StatelessWidget {
       Events model, int index, String eventDocId, BuildContext context) {
     return InkWell(
       onTap: () => onTapItem(model, eventDocId, context),
-      child: PrimaryContainerEvents(index: index, modelEvents: model),
+      child: primaryContainer(index, model),
     );
   }
 
+  PrimaryContainer primaryContainer(int index, Events model) {
```

Comment:
Misleading name. Better something like "EventContainer", but this is more than a container.

Reply:
solved #14